env.file

```diff
@@ -2,3 +2,27 @@ SERVER_URL=http://localhost/
 DOMAIN_REDIRECT_URL=http://localhost:3000
 WEB_OIDC_CLIENT_ID=teammail-web
 APP_GRID_AVAILABLE=supported
+
+FIREBASE_ANDROID_API_KEY=AIzaSyDPn3T4BMShrcedtqzA-8Lh3Fsm1EGB7S8
+FIREBASE_ANDROID_APP_ID=1:611786135906:android:6fa64d94eb8f8603e3743b
+FIREBASE_ANDROID_MESSAGING_SENDER_ID=611786135906
+FIREBASE_ANDROID_PROJECT_ID=fir-jmap-poc
+FIREBASE_ANDROID_DATABASE_URL=https://fir-jmap-poc-default-rtdb.asia-southeast1.firebasedatabase.app
+FIREBASE_ANDROID_STORAGE_BUCKET=fir-jmap-poc.appspot.com
+
+FIREBASE_IOS_API_KEY=AIzaSyA_fAC-6rVLsQtj8AXxRRYjwsOHCKPTm6w
+FIREBASE_IOS_APP_ID=1:611786135906:ios:d8b0899dc058dd2ae3743b
+FIREBASE_IOS_MESSAGING_SENDER_ID=611786135906
+FIREBASE_IOS_PROJECT_ID=fir-jmap-poc
+FIREBASE_IOS_DATABASE_URL=https://fir-jmap-poc-default-rtdb.asia-southeast1.firebasedatabase.app
+FIREBASE_IOS_STORAGE_BUCKET=fir-jmap-poc.appspot.com
+FIREBASE_IOS_CLIENT_ID=611786135906-2nqn5c537eer6c95rcujmlfmldbkqfqb.apps.googleusercontent.com
+FIREBASE_IOS_BUNDLE_ID=com.linagora.ios.teammail
+
+FIREBASE_WEB_API_KEY=AIzaSyABpxntIXA2u_uFFMT4wvHYMgVm7UZPMrM
+FIREBASE_WEB_APP_ID=1:611786135906:web:22d98cfd47b4fddde3743b
+FIREBASE_WEB_MESSAGING_SENDER_ID=611786135906
+FIREBASE_WEB_PROJECT_ID=fir-jmap-poc
+FIREBASE_WEB_DATABASE_URL=https://fir-jmap-poc-default-rtdb.asia-southeast1.firebasedatabase.app
+FIREBASE_WEB_STORAGE_BUCKET=fir-jmap-poc.appspot.com
+FIREBASE_WEB_AUTH_DOMAIN=fir-jmap-poc.firebaseapp.com
```

Comment:
Be careful!!!!

Reply:
I don't pay much attention to it. We will discuss further with @manh